### DISCUSSION TITLE
Attaching components

### DISCUSSION THREAD

**Discussion by vanhound** [atteneder/glTFast#326] [Ideas]
Is it possible to attach components to new instantiated GameObjects through  `gltf.InstantiateMainScene(new GameObject("PlacedModel").transform);` ?

**Comment by atteneder** [comment]
@Alfcovap51 Can you elaborate a bit more? I don't quite understand the question.

Of course you can make modifications to GameObjects and its components after the instantiation.

If you want to add components during instantiation based on a certain heuristic, then please explain your use-case in detail so we can reason about it.

Thanks
